.changeset/nfc-payment-icon.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': minor
+---
+
+Added 24px NFC payment icon in Product and Features
```

Comment:
Can you also update the changeset accordingly.

Reply:
thanks! done 👌